test/OpenTelemetry.Tests.Stress/OpenTelemetry.Tests.Stress.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0;net462</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1;net462</TargetFrameworks>
```

Comment:
Why do we have these changes? Is similar thing guarded by the CI in the future? (if not, I think it could run out of sync across files)

Reply:
Based on the PR description, I'm guessing this is to workaround OmniSharp on VS Mac?

Reply:
That's exactly what it is. OmniSharp. It's not specific to Mac - it's a general OmniSharp problem on all platforms.

Reply:
Maybe add a comment above the targetframeworks tag to state the need to have this specific order, so none accidentally reorders this.
(Eventually we need to move it to common place, but thats definitely a separate PR.)

Reply:
Here is my guess - without proper CI in place, it is very likely that other contributors might break it again - requiring the maintainers to pay close attention to such details in every single PR is unlikely to sustain.

Consider adding a simple script to validate these or having some OmniSharp CI job https://github.com/open-telemetry/opentelemetry-dotnet/blob/5bb9f26100a0fff8996c0b861b12e27bcf513f79/build/sanitycheck.py.

Reply:
a CI job validating this is definitely safer, but a simple comment would also work - its not common for PRs to touch target frameworks.

Reply:
I added a comment to every `TargetFrameworks` node:

`<!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->`

Admittedly, this is _technically inaccurate_: it picks the first one listed. However, it also seemed like having the frameworks listed in a sensible/predictable order was valuable; and it didn't seem like writing a giant paragraph explaining the technical ins-and-outs of the overlap of OmniSharp and ordering was valuable.

However, if you have preferred alternate verbiage, I can run a giant search and replace on the existing comment and update accordingly.

Reply:
this looks reasonable.